zaza/openstack/charm_tests/ceph/dashboard/tests.py

```diff
@@ -20,6 +20,8 @@
 import requests
 import tenacity
 import uuid
+import trustme
+from base64 import b64encode
```

Comment:
Nit: I realize this wasn't very tidy before but it would be nice to get import ordering conform to PEP8, i.e.:
- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

cf. https://peps.python.org/pep-0008/#imports

Reply:
ack